inductiva/coastal/README.md

```diff
@@ -0,0 +1,93 @@
+# Coastal dynamics
+
+Inductiva API supports simuulation of coastal dynamics via two simulators:
+[SWASH](https://swash.sourceforge.io/) and [XBeach](https://oss.deltares.nl/web/xbeach/).
+These simulators solve the [shallow-water equations](https://en.wikipedia.org/wiki/Shallow_water_equations),
+a reduced version of the Navier-Stokes equations valid for small depth-to-length
+ratios (i.e., shallow water).
+
```

Comment:
You can invoke each of these simulators directly to execute your own simulation scripts, or you can make use of the CoastalArea scenario that provides you a high-level interface for simulating wave conditions on  bathymetries you can load.

Invoking SWASH or XBeach Directly
[short example. We had it somewhere already...]


[try to expand to something along these line]

Reply:
Because we have it already somewhere, I think we shouldn't repeat it here. WDYT @sarmento?

Reply:
I think it is ok to repeat it here, to allow this to become more self-contained. Once this is self-contained, then we can point people interested in coastal dynamics directly to this page.